### PR TITLE
fix navtree deprecations 

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
Can you please split this into one commit with black/isort and one with the interesting changes?  That is much easier to review.  Now it is not clear at all what the actually interesting changes are.

File: Products/CMFPlone/browser/navtree.py [hunk 1]
```diff
@@ -108,84 +107,80 @@ class SitemapNavtreeStrategy(NavtreeStrategyBase):
...
+        newNode["getURL"] = itemUrl
+        newNode["path"] = item.getPath()
+        newNode["Creator"] = getattr(item, "Creator", None)
+        newNode["creation_date"] = getattr(item, "CreationDate", None)
+        newNode["portal_type"] = portalType
+        newNode["review_state"] = getattr(item, "review_state", None)
+        newNode["Description"] = getattr(item, "Description", None)
+        newNode["show_children"] = showChildren
+        newNode["no_display"] = False  # We sort this out with the nodeFilter
         # BBB getRemoteUrl and link_remote are deprecated, remove in Plone 4
```
Comment (gforcada) [inline Products/CMFPlone/browser/navtree.py:166]: 😆 we are a bit late at deprecating these `BBB` comment 😅

**Review by gforcada — APPROVED** [review]
It would be nice if you can structure your commits with automatic changes (either first or last) and then the actual changes, that would make reviewing much easier ✨

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**